### ISSUE TITLE
Feature Request: supply .pac file via REST server

### ISSUE THREAD

**Issue by echenley** [SpectoLabs/hoverfly#881]
I am very new to Hoverfly, so apologies if this request is mildly incoherent. However, I could not find any existing issues that seemed relevant.

**Is your feature request related to a problem? Please describe.**

At my company, we configure our corporate proxy using a .pac file. [This PR](https://github.com/SpectoLabs/hoverfly/pull/766) added the ability to configure an upstream proxy via a pac file, which is great! However, in order to route requests in the browser (Chrome) through the hoverfly proxy, I need to update my proxy configuration via Mac network settings. Since we already use Automatic Proxy Configuration via a .pac file, it would be especially convenient to just switch to a .pac file created by the Hoverfly instance, rather than changing the proxy configuration method.

**Describe the solution you'd like**

Basically this:

![Screen Shot 2019-10-28 at 5 14 59 PM](https://user-images.githubusercontent.com/6344219/67722216-830c0080-f9a6-11e9-82b1-20b6a1245b07.jpg)

**Comment by echenley** [comment]
Found a way around this by opening chrome programmatically with the --proxy-server flag, which does not require changing system proxy settings.

```sh
open -na "Google Chrome" --args --proxy-server=http://localhost:8500
```

**Comment by JohnFDavenport** [comment]
Thanks for letting us know @echenley. I will leave the issue open for further comment for the time-being.
